docs/fundamentals/toc.yml

```diff
@@ -934,22 +934,24 @@ items:
             href: ../standard/commandline/index.md
           - name: Get started tutorial
             href: ../standard/commandline/get-started-tutorial.md
-          - name: Command-line syntax
+          - name: Command-line syntax overview
             href: ../standard/commandline/syntax.md
-          - name: Define commands
-            href: ../standard/commandline/define-commands.md
-          - name: Model binding
-            href: ../standard/commandline/model-binding.md
-          - name: Tab completion
-            href: ../standard/commandline/tab-completion.md
-          - name: Dependency injection
-            href: ../standard/commandline/dependency-injection.md
-          - name: Customize help
-            href: ../standard/commandline/customize-help.md
-          - name: Handle termination
-            href: ../standard/commandline/handle-termination.md
-          - name: Use middleware
-            href: ../standard/commandline/use-middleware.md
+          - name: How to
+            items:
+            - name: Parse and invoke the result
```

Comment:
Also, with all of these articles here under a **How to** section, if they're really how-to articles, it helps SEO if their file name is `how-to-*.md`.